collectors/bzimport/tests/test_convertors.py

```diff
@@ -43,6 +43,7 @@ def get_flaw(self):
             title="title",
             description="description",
             impact=Impact.CRITICAL,
+            major_incident_state="REQUESTED",
```

Comment:
Use enums (eg. `Flaw.FlawMajorIncident.REQUESTED`) instead of string if possible.